bot.py

```diff
@@ -155,13 +155,15 @@ async def remove(ctx):
 @bot.command()
 async def inv(ctx):
     author = ctx.message.author.id
-    await ctx.message.channel.send(f"Current user is {author}")
+    name = ctx.message.author.name
+    # For testing
+    # await ctx.message.channel.send(f"Current user is {author}")
```

Comment:
```suggestion
```

Leaving this comment behind isn't helpful. This can easily grow stale. We should write tests instead of leaving test comments behind.